src/huggingface_hub/inference/_client.py

```diff
@@ -348,7 +348,6 @@ def audio_classification(
             top_k (`int`, *optional*):
                 When specified, limits the output to the top K most probable classes.
             function_to_apply (`"AudioClassificationOutputTransform"`, *optional*):
-                The function to apply to the output.
```

Comment:
I'll open a PR to add the docstring of `function_to_apply` in huggingface.js

Reply:
[huggingface/huggingface.js/pull/1043](https://github.com/huggingface/huggingface.js/pull/1043)